tide/src/lib.rs

```diff
@@ -28,27 +25,19 @@ macro_rules! box_async {
 }
 
 #[macro_use]
-pub mod error;
+extern crate tide_core;
+
+#[cfg(feature = "cookies")]
+pub use tide_cookies as cookies;
 
-mod app;
-mod context;
-pub mod cookies;
-mod endpoint;
+pub mod error;
 pub mod forms;
 pub mod middleware;
 pub mod querystring;
-pub mod response;
-mod route;
-mod router;
+
+pub use tide_core::response;
```

Comment:
Can we add `#[doc(inline)]` above all `tide_core` re-exports?

Reply:
Or better: this could probably be part of line 44.

Reply:
I've somehow missed this.